extensions/amp-analytics/0.1/analytics-root.js

```diff
@@ -210,20 +210,24 @@ export class AnalyticsRoot {
       let found;
       let result = null;
       // Query search based on the selection method.
-      if (selectionMethod == 'scope') {
-        found = scopedQuerySelector(context, selector);
-      } else if (selectionMethod == 'closest') {
-        found = closestBySelector(context, selector);
-      } else {
-        found = this.getRoot().querySelector(selector);
-      }
-      // DOM search can "look" outside the boundaries of the root, thus make
-      // sure the result is contained.
-      if (found && this.contains(found)) {
-        result = found;
+      try {
+        if (selectionMethod == 'scope') {
+          found = scopedQuerySelector(context, selector);
+        } else if (selectionMethod == 'closest') {
+          found = closestBySelector(context, selector);
+        } else {
+          found = this.getRoot().querySelector(selector);
+        }
+        // DOM search can "look" outside the boundaries of the root, thus make
+        // sure the result is contained.
+        if (found && this.contains(found)) {
+          result = found;
+        }
+        return user().assertElement(
```

Comment:
we can move this to be after the catch block to avoid doubling the error messages.

Reply:
done